msmtools/analysis/dense/gpcca.py

```diff
@@ -1075,10 +1075,10 @@ def __init__(self, P, eta=None, z='LM', method='brandts'):
         if method not in ['brandts', 'krylov']:
             raise ValueError("You didn't give a valid method to determine the invariant subspace.")
           
-        if issparse(P) and method != 'krylov':
```

Comment:
I just recognized that in `gpcca_coarsegrain` sparse matrices are not handled...

Reply:
For now, I fixed this by just making it dense in case it's sparse. I added a todo and will open an issue for this, so we can do it properly it a separate PR.